reference/requirements.rst

```diff
@@ -7,11 +7,15 @@ Requirements for Running Symfony
 ================================
 
 Symfony 3.4 requires **PHP 5.5.9** or higher to run, in addition to other minor
-requirements. To make things simple, Symfony provides a tool to quickly check if
-your system meets all those requirements.
+requirements, when using the traditional installation based on the
+`Symfony standard edition`_. If Symfony 3.4 is installed via the `skeleton`_ or
```

Comment:
Should be „Symfony Standard Edition“ die to:
https://github.com/symfony/symfony-standard